nautobot/core/views/__init__.py

```diff
@@ -181,6 +182,9 @@ def server_error(request, template_name=ERROR_500_TEMPLATE_NAME):
 
 
 class CustomGraphQLView(GraphQLView):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs, schema=generate_schema())
```

Comment:
Generating the schema seems pretty slow - does this now make it so that we have that delay each and every time the GraphQL API is accessed?

Actually, come to think of it - this view is specifically for GraphiQL - the actual non-interactive GraphQL endpoint is at GraphQLDRFAPIView in nautobot/core/api/views.py, doesn't that need to be changed as well?